### PR TITLE
docs: update Prometheus tutorial

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendu@apache.org>

Updates the tutorial on using Prometheus with Apache APISIX Ingress.

### CODE REVIEW

File: docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 1]
```diff
@@ -21,24 +26,21 @@ title: How to access Apache APISIX Prometheus metrics on Kubernetes
...
-```sh
-helm repo add apisix https://charts.apiseven.com
-helm repo update
-helm install apisix apisix/apisix -f values.yaml --create-namespace -n ingress-apisix
-```
+## Enable the prometheus Plugin
 
-<details>
-  <summary>values.yaml</summary>
+First, you have to enable the [prometheus](https://apisix.apache.org/docs/apisix/plugins/prometheus) Plugin. You can do this by adding to your `values.yaml` file while you install APISIX Ingress via Helm. You can also enable the [public-api](https://apisix.apache.org/docs/apisix/next/plugins/public-api) Plugin to expose these metrics.
```
Comment (juzhiyuan) [inline docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:39]: Can we remove the "next" prefix in documentation url?

**Review by juzhiyuan — COMMENTED** [review]
Others LGTM

File: docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 1 above)
Comment (navendu-pottekkat) [reply docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:39]: Yes. I always forget it.

File: docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 2]
```diff
@@ -129,16 +131,11 @@ spec:
       enable: true
 ```
 
-### Step 2: Enable `public-api` Plugin
+## Expose metrics with the public-api Plugin
 
-Let's make a basic routing setup, and please note that further configuration should be done based on your local backend service information. The primary solution concept is to use the `public-api` plugin to protect the routes exposed by *Prometheus*. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
-
-```bash
-kubectl apply -f prometheus-route.yaml -n ingress-apisix
```
Comment (AlinsRan) [inline docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:137]: If you want to delete this line, please add the description of the specified namespace. Or specify the namespace in yaml.

**Review by AlinsRan — CHANGES_REQUESTED** [review]

File: docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 2 above)
Comment (AlinsRan) [reply docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:137]: Have you tried this tutorial?

Comment (navendu-pottekkat) [reply docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:137]: > Have you tried this tutorial?

Yes. I always try the tutorial/docs pages to make sure that there are no errors or if we need to make any updates.

> If you want to delete this line, please add the description of the specified namespace. Or specify the namespace in yaml.

Yes. Makes sense. Explicitly mentioning the namespace is a good idea. I will add it.

Comment (navendu-pottekkat) [reply docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:137]: @AlinsRan Is it okay now?

**Review by AlinsRan — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**